sdk/include/opentelemetry/sdk/logs/batch_log_record_processor.h

```diff
@@ -115,20 +115,25 @@ class BatchLogRecordProcessor : public LogRecordProcessor
 
     /* Important boolean flags to handle the workflow of the processor */
     std::atomic<bool> is_force_wakeup_background_worker{false};
-    std::atomic<bool> is_force_flush_pending{false};
-    std::atomic<bool> is_force_flush_notified{false};
-    std::atomic<std::chrono::microseconds::rep> force_flush_timeout_us{0};
     std::atomic<bool> is_shutdown{false};
+    std::atomic<std::uint64_t> force_flush_pending_sequence{false};
```

Comment:
nit: initialize std::atomic<uint64_t> variable to 0 instead of `false`.

Reply:
Thanks and fixed.